awscli/examples/devicefarm/list-devices.rst

```diff
@@ -0,0 +1,27 @@
+### To list all iOS private devices and their UDID value
+
+```bash
+aws devicefarm list-devices --region us-west-2 --query 'devices[?platform==`IOS` && fleetType==`PRIVATE`].[name, os, instances[].udid]'
```

Comment:
We can probably remove the `--region us-west-2` to maintain consistency between all the different service command/subcommand examples (we'll just assume that the user has already configured their region somewhere else, such as in their `~/.aws/config` or via an environment variable).